doc/source/examples/analysis/custom_parallel_analysis.ipynb

```diff
@@ -10,9 +10,9 @@
     "\n",
```

Comment:
<p>setting n_jobs to cpu_count is going to lead to this changing between machines (since folks will have different machines). Would it be best to just hardcode it to a given value?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
given that the notebook needs to run on the instance, 2 is the only real option.
I'd still leave this commented out
```
from multiprocessing import cpu_count
n_jobs = cpu_count()
```
since it's useful for the user.